src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -197,18 +200,18 @@
       [quo/input
        {:type              :password
         :blur?             true
-        :override-theme    :dark
+        :disabled?         processing
         :placeholder       (i18n/label :t/type-your-password)
         :auto-focus        true
-        :error?            (when (not-empty error) error)
+        :error?            (seq error)
         :label             (i18n/label :t/profile-password)
-        :secure-text-entry true
         :on-change-text    (fn [password]
-                             (rf/dispatch [:set-in [:multiaccounts/login :password]
-                                           (security/mask-data password)])
+                             (rf/dispatch-sync [:set-in [:multiaccounts/login :password]
+                                                (security/mask-data password)])
                              (rf/dispatch [:set-in [:multiaccounts/login :error] ""]))
+        :value             (security/safe-unmask-data password)
```

Comment:
Related to this {PR main purpose, I added the `:value` key so the value is not lost.
But also had to use dispatch-sync since we want to use re-frame for an input, if we don't use dispatch-sync, the input will flicker.

@Parveshdhull

Reply:
something is implemented wrong then, we need to check, but `dispatch-sync` shouldn't be used

Reply:
Hmm... Why should it be used? even it's recommended in re-frame for this use case

https://day8.github.io/re-frame/api-re-frame.core/#dispatch-sync

> Generally, avoid using this function, and instead, use dispatch. Only use it in the narrow set of cases where any delay in processing is a problem:
>
  > 1.  the :on-change handler of a text field where we are expecting fast typing
  > 2.  ...

Reply:
:value shouldn't be used, :default-value should be used instead, we don't want to entered-text-> mask->unmask->replace just entered text , doesn't make sense

Reply:
btw for password why would we want to have default value at all ?

Reply:
I think this mask->unmask is due to the decision of using re-frame to store an input's value.

I think using `default-value` should work, let me try

Reply:
> I think using `default-value` should work, let me try

i don't think its needed, just remove :value at all

Reply:
please see my comments above

Reply:
> btw for password why would we want to have default value at all ?

When we disable the input component, (we disable it when the user logs-in), it is re-initialized so the value in the input is erased, I added the `:default-value` key to keep the value